src/components/pages/Index.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import AddFarmer from '../pages/AddFarmer/AddFarmer.jsx'
 
 const Index = () => {
   return (
     <header>
       <h1>Tieme Ndo Home</h1>
+      <AddFarmer/>
     </header>
   );
```

Comment:
Please merge with master and include AddFarmer component inside a Route

Reply:
its just a draft pr